src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -0,0 +1,83 @@
+* Check the [Sync Logs](https://pontoon.mozilla.org/sync/log/) page, click on the last one, and make sure that there are no projects with `---` in the `DURATION` field.
+* In [Heroku](https://dashboard.heroku.com/apps/mozilla-pontoon/resources), switch the *worker* from `PM` (Performance-M dynos) to `PL` (Performance-L dynos).
+
+It’s important to make sure that there is no sync in progress when upgrading the worker, because that will kill any pending process.
+
+## Push updates to gecko-strings
+
+Assuming the new strings are available in the [quarantine repository](https://hg.mozilla.org/users/axel_mozilla.com/gecko-strings-quarantine), it’s now time to push the updates to the official [gecko-strings repository](https://hg.mozilla.org/l10n/gecko-strings).
+
+Pontoon relies on a [different repository](https://hg.mozilla.org/users/m_owca.info/firefox-central/), that is generated every 20 minutes based on the content of `gecko-strings`. That’s why it’s useful to run this update before starting the actual migrations.
+
+## Run migrations
+
+A series of tools and helpers to run migrations is available in [this repository](https://github.com/flodolo/fluent-migrations):
+* Clone the repository locally and follow the instructions available in the README to set the config.
+* Add the new migration recipe in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder.
+
+At this point, while the migration has been tested as part of the review before landing, it’s always good to run the migration against one locale, e.g. `it`:
+* Update the locale’s repository, check the results of `compare-locales`. Assuming l10n repositories are cloned in `~/migrations/locales`, and the quarantine repository in `~/migrations/gecko-strings-quarantine`, the command to run is `compare-locales ~/migrations/gecko-strings-quarantine/_configs/browser.toml ~/migrations/locales it`. Save the output in order to compare it with the results after the migration.
+* Run the migration, without pushing, only for `it`: `./scripts/migrate it wet-run`. Then run compare-locales again, and check if the results are as expected: migrated strings should not appear as missing anymore, and there should be no errors.
+* If everything looks good, re-run the script adding the push option `./scripts/migrate it wet-run push`.
+
+To run the migration on all repositories, use `./scripts/migrate wet-run push`. For each repository, this will:
+* Pull changes.
+* Run the migration and commit changes.
+* Push changes to the remote.
+
+To run multiple migrations, put all recipes in the `recipes` folder.
+
+The duration of this step depends on the number of migrations and the speed of the Internet connection, but it should take about 20-30 minutes. While migrations run, it’s important to look out for errors in the console (e.g. hg or Python errors).
```

Comment:
Are there any tips what to do when errors show up?

Reply:
I haven't seen Python errors in a long time, because the code is more resilient. It's more frequent to see hg errors. Adding a note anyway.